docs/decisions/README.md

```diff
@@ -0,0 +1,22 @@
+# Markdown Any Decision Records
+
+MADR is a lean template to capture any decisions in a structured way. The template originated from capturing architectural decisions and developed to a template allowing to capture any decisions taken.
+For more information [see](https://adr.github.io/madr/)
+
+## How are we using ADR's to track technical decisions?
+
+1. Copy docs/decisions/adr-template.md to docs/decisions/NNNN-title-with-dashes.md, where NNNN indicates the next number in sequence.
+    1. Check for existing PR's to make sure you use the correct sequence number.
+    2. There is also a short form template docs/decisions/adr-short-template.md
+2. Edit NNNN-title-with-dashes.md.
+    1. Status must initially be `proposed`
+    2. List of `deciders` must include the aliases of the people who will sign off on the decision.
+    3. The relevant EM and `dluc` must be listed as deciders or informed of all decisions.
```

Comment:
This is not necessarily true for all designs.

Reply:
perhaps:

```suggestion
    3. The relevant EM and architect must be listed as deciders or informed of all decisions.
```

Reply:
Done